src/System.Private.CoreLib/shared/System/Runtime/Intrinsics/X86/Avx2.cs

```diff
@@ -546,243 +546,963 @@ public abstract class Avx2 : Avx
         /// __m128i _mm_i32gather_epi32 (int const* base_addr, __m128i vindex, const int scale)
         ///   VPGATHERDD xmm, vm32x, xmm
         /// </summary>
-        public static unsafe Vector128<int> GatherVector128(int* baseAddress, Vector128<int> index, byte scale) => GatherVector128(baseAddress, index, scale);
+        public static unsafe Vector128<int> GatherVector128(int* baseAddress, Vector128<int> index, byte scale)
+        {
```

Comment:
Perhaps we should apply `AggressiveInlining` attribute to all managed implementations

Reply:
No, the fallback should always be a call-node.